### PR TITLE
chore(cnpg-pooler): re-apply the pooler urls

### PR DESCRIPTION
**Pull request by stavros-k**
**Description**

⚒️ Fixes  # 

Re-applies this https://github.com/truecharts/library-charts/commit/988d82f668e91274ace8a35b0f6dc8b16dde20eb

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**

**📃 Notes:**

**✔️ Checklist:**

- [x] ⚖️ My code follows the style guidelines of this project
- [x] 👀 I have performed a self-review of my own code
- [x] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [x] ⚠️ My changes generate no new warnings
- [x] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [x] ⬆️ I increased versions for any altered app according to semantic versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

### CODE REVIEW

**Comment by Ornias1993** [comment]
@stavros-k solution:
If Pooler == enabled
then use pooler url for secret, otherwise use cluster url for secret.

Easy fix and would mean we can fix this relatively quickly

**Comment by stavros-k** [comment]
Isn't that the case already? But even if it isn't,  This wasn't the actual problem that made us revert.
The pooler was running but still connection was not made.

**Comment by Ornias1993** [comment]
> Isn't that the case already? But even if it isn't, This wasn't the actual problem that made us revert. The pooler was running but still connection was not made.

What I meant was to change the urls in the secret dynamically.

**Comment by Ornias1993** [comment]
@stavros-k Updated PR to include my changes

File: library/common/templates/lib/cnpg/_dbCredentialSecrets.tpl [hunk 1]
```diff
@@ -13,22 +13,29 @@
   {{- $hostTmpl := printf "%s-%%s" $objectData.name -}}
   {{- $jdbcTmpl := printf "jdbc:postgresql://%s-%%s:5432/%s" $objectData.name $objectData.database -}}
 
+  {{- $rwString := "rw" -}}
+  {{- $roString := "ro" -}}
+  {{- if $objectData.pooler.enabled -}}
```
Comment (Ornias1993) [inline library/common/templates/lib/cnpg/_dbCredentialSecrets.tpl:18]:
```suggestion
  {{- if and $objectData.pooler $objectData.pooler.enabled -}}
```

**Comment by stavros-k** [comment]
Should be green now

**Pull request merged**